_docs-2/getting-started/features.md

```diff
@@ -16,23 +16,23 @@ redirect_from:
 
 ### Iterators
 
-A server-side programming mechanism to encode functions such as filtering and
+[Iterators] are a server-side programming mechanism that encode functions such as filtering and
```

Comment:
I think you are mixing plural and single phrases... Should say "An iterator is a server-side mechanism that encodes..." or "Iterators are server-side programming mechanisms that encode"